### ISSUE TITLE
Error initializing complex filters. Numerical result out of range

### ISSUE THREAD

**Issue by danisham-afk** [slhck/ffmpeg-normalize#110]
**Expected behavior**
I'm trying to process file https://commons.wikimedia.org/wiki/File:Ru-%D0%90%D0%BD%D0%B3%D0%BE%D0%BB%D0%B0.ogg

**Actual behavior**
Error  Numerical result out of range

**Command**
```
ffmpeg-normalize Ru-Ангола.ogg -prf atempo=1 -c:a libmp3lame -b:a 320k -ext mp3
```

Any output you get when running the command with the `--debug` flag:

```
ffmpeg-normalize Ru-Ангола.ogg -prf atempo=1 -t -20 -c:a libmp3lame -b:a 320k -ext mp3 --debug
DEBUG: found executable in path: /usr/bin/ffmpeg
DEBUG: found executable in path: /usr/bin/ffmpeg
DEBUG: Running command: ['/usr/bin/ffmpeg', '-filters']
DEBUG: Parsing streams of Ru-Ангола.ogg
DEBUG: Running command: ['/usr/bin/ffmpeg', '-i', 'Ru-Ангола.ogg', '-c', 'copy', '-t', '0', '-map', '0', '-f', 'null', '/dev/null']
DEBUG: Stream parsing command output:
DEBUG: ffmpeg version 4.2.1-0york0~18.04.1 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 7 (Ubuntu 7.4.0-1ubuntu1~18.04.1)
  configuration: --prefix=/usr --extra-version='0york0~18.04.1' --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-nonfree --enable-libfdk-aac --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
Input #0, ogg, from 'Ru-Ангола.ogg':
  Duration: 00:00:01.20, start: 0.000000, bitrate: 131 kb/s
    Stream #0:0: Audio: vorbis, 48000 Hz, stereo, fltp, 128 kb/s
    Metadata:
      DATE            : 2013-10-13
      ENGINEER        : Bar Barov
Output #0, null, to '/dev/null':
  Metadata:
    encoder         : Lavf58.29.100
    Stream #0:0: Audio: vorbis, 48000 Hz, stereo, fltp, 128 kb/s
    Metadata:
      DATE            : 2013-10-13
      ENGINEER        : Bar Barov
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
Press [q] to stop, [?] for help
size=N/A time=-00:00:00.00 bitrate=N/A speed=N/A    
video:0kB audio:0kB subtitle:0kB other streams:0kB global headers:4kB muxing overhead: unknown

DEBUG: Found duration: 1.02 s
DEBUG: Found audio stream at index 0
WARNING: Audio stream has a duration of less than 3 seconds. Normalization may not work. See https://github.com/slhck/ffmpeg-normalize/issues/87 for more info.
INFO: Normalizing file Ru-Ангола.ogg (1 of 1)
DEBUG: Running normalization for Ru-Ангола.ogg
DEBUG: Parsing normalization info for Ru-Ангола.ogg
INFO: Running first pass loudnorm filter for stream 0
DEBUG: Running ffmpeg command: ['/usr/bin/ffmpeg', '-nostdin', '-y', '-i', 'Ru-Ангола.ogg', '-filter_complex', '[0:0]loudnorm=i=-20.0:lra=7.0:tp=-2.0:offset=0.0:print_format=json', '-vn', '-sn', '-f', 'null', '/dev/null']
DEBUG: Loudnorm first pass command output:
DEBUG: ffmpeg version 4.2.1-0york0~18.04.1 Copyright (c) 2000-2019 the FFmpeg developers
built with gcc 7 (Ubuntu 7.4.0-1ubuntu1~18.04.1)
configuration: --prefix=/usr --extra-version='0york0~18.04.1' --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-nonfree --enable-libfdk-aac --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
libavutil      56. 31.100 / 56. 31.100
libavcodec     58. 54.100 / 58. 54.100
libavformat    58. 29.100 / 58. 29.100
libavdevice    58.  8.100 / 58.  8.100
libavfilter     7. 57.100 /  7. 57.100
libavresample   4.  0.  0 /  4.  0.  0
libswscale      5.  5.100 /  5.  5.100
libswresample   3.  5.100 /  3.  5.100
libpostproc    55.  5.100 / 55.  5.100
Input #0, ogg, from 'Ru-Ангола.ogg':
Duration: 00:00:01.20, start: 0.000000, bitrate: 131 kb/s
Stream #0:0: Audio: vorbis, 48000 Hz, stereo, fltp, 128 kb/s
Metadata:
DATE            : 2013-10-13
ENGINEER        : Bar Barov
Stream mapping:
Stream #0:0 (vorbis) -> loudnorm
loudnorm -> Stream #0:0 (pcm_s16le)
Output #0, null, to '/dev/null':
Metadata:
encoder         : Lavf58.29.100
Stream #0:0: Audio: pcm_s16le, 192000 Hz, stereo, s16, 6144 kb/s
Metadata:
encoder         : Lavc58.54.100 pcm_s16le
size=N/A time=00:00:01.22 bitrate=N/A speed=36.8x
bitrate=N/A
total_size=N/A
out_time_us=1222667
out_time_ms=1222667
out_time=00:00:01.222667
dup_frames=0
drop_frames=0
speed=36.8x
progress=end
video:0kB audio:917kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[Parsed_loudnorm_0 @ 0x556cb62c7ec0]
{
"input_i" : "29.95",
"input_tp" : "62.45",
"input_lra" : "0.00",
"input_thresh" : "10.41",
"output_i" : "-inf",
"output_tp" : "-1263.14",
"output_lra" : "0.00",
"output_thresh" : "-70.00",
"normalization_type" : "linear",
"target_offset" : "inf"
}




DEBUG: Loudnorm stats parsed: {"input_i": "29.95", "input_tp": "62.45", "input_lra": "0.00", "input_thresh": "10.41", "output_i": "-inf", "output_tp": "-1263.14", "output_lra": "0.00", "output_thresh": "-70.00", "normalization_type": "linear", "target_offset": "inf"}
INFO: Running second pass for Ru-Ангола.ogg
WARNING: Input file had measured input loudness greater than zero (input_i), capping at 0
DEBUG: Running ffmpeg command: ['/usr/bin/ffmpeg', '-y', '-nostdin', '-i', 'Ru-Ангола.ogg', '-filter_complex', '[0:0]atempo=1,loudnorm=i=-20.0:lra=7.0:tp=-2.0:offset=0.0:measured_i=0.0:measured_lra=0.0:measured_tp=62.45:measured_thresh=10.41:linear=true:print_format=json[norm0]', '-map_metadata', '0', '-map_metadata:s:a:0', '0:s:a:0', '-map_chapters', '0', '-c:v', 'copy', '-map', '[norm0]', '-c:a', 'libmp3lame', '-b:a', '320k', '-c:s', 'copy', '/tmp/i85ob8e8.mp3']
ERROR: Error while running command ['/usr/bin/ffmpeg', '-y', '-nostdin', '-i', 'Ru-Ангола.ogg', '-filter_complex', '[0:0]atempo=1,loudnorm=i=-20.0:lra=7.0:tp=-2.0:offset=0.0:measured_i=0.0:measured_lra=0.0:measured_tp=62.45:measured_thresh=10.41:linear=true:print_format=json[norm0]', '-map_metadata', '0', '-map_metadata:s:a:0', '0:s:a:0', '-map_chapters', '0', '-c:v', 'copy', '-map', '[norm0]', '-c:a', 'libmp3lame', '-b:a', '320k', '-c:s', 'copy', '/tmp/i85ob8e8.mp3']!
Traceback (most recent call last):
  File "/home/user/projects/test/env/bin/ffmpeg-normalize", line 10, in <module>
    sys.exit(main())
  File "/home/user/projects/test/env/lib/python3.6/site-packages/ffmpeg_normalize/__main__.py", line 406, in main
    ffmpeg_normalize.run_normalization()
  File "/home/user/projects/test/env/lib/python3.6/site-packages/ffmpeg_normalize/_ffmpeg_normalize.py", line 180, in run_normalization
    media_file.run_normalization()
  File "/home/user/projects/test/env/lib/python3.6/site-packages/ffmpeg_normalize/_media_file.py", line 146, in run_normalization
    for _ in self._second_pass():
  File "/home/user/projects/test/env/lib/python3.6/site-packages/ffmpeg_normalize/_media_file.py", line 335, in _second_pass
    raise e
  File "/home/user/projects/test/env/lib/python3.6/site-packages/ffmpeg_normalize/_media_file.py", line 321, in _second_pass
    for progress in cmd_runner.run_ffmpeg_command():
  File "/home/user/projects/test/env/lib/python3.6/site-packages/ffmpeg_normalize/_cmd_utils.py", line 89, in run_ffmpeg_command
    raise RuntimeError("Error running command {}: {}".format(self.cmd, str("\n".join(stderr))))
RuntimeError: Error running command ['/usr/bin/ffmpeg', '-y', '-nostdin', '-i', 'Ru-Ангола.ogg', '-filter_complex', '[0:0]atempo=1,loudnorm=i=-20.0:lra=7.0:tp=-2.0:offset=0.0:measured_i=0.0:measured_lra=0.0:measured_tp=62.45:measured_thresh=10.41:linear=true:print_format=json[norm0]', '-map_metadata', '0', '-map_metadata:s:a:0', '0:s:a:0', '-map_chapters', '0', '-c:v', 'copy', '-map', '[norm0]', '-c:a', 'libmp3lame', '-b:a', '320k', '-c:s', 'copy', '/tmp/i85ob8e8.mp3']: ffmpeg version 4.2.1-0york0~18.04.1 Copyright (c) 2000-2019 the FFmpeg developers
built with gcc 7 (Ubuntu 7.4.0-1ubuntu1~18.04.1)
configuration: --prefix=/usr --extra-version='0york0~18.04.1' --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-nonfree --enable-libfdk-aac --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
libavutil      56. 31.100 / 56. 31.100
libavcodec     58. 54.100 / 58. 54.100
libavformat    58. 29.100 / 58. 29.100
libavdevice    58.  8.100 / 58.  8.100
libavfilter     7. 57.100 /  7. 57.100
libavresample   4.  0.  0 /  4.  0.  0
libswscale      5.  5.100 /  5.  5.100
libswresample   3.  5.100 /  3.  5.100
libpostproc    55.  5.100 / 55.  5.100
Input #0, ogg, from 'Ru-Ангола.ogg':
Duration: 00:00:01.20, start: 0.000000, bitrate: 131 kb/s
Stream #0:0: Audio: vorbis, 48000 Hz, stereo, fltp, 128 kb/s
Metadata:
DATE            : 2013-10-13
ENGINEER        : Bar Barov
[loudnorm @ 0x5610df04ac80] Value 10.410000 for parameter 'measured_thresh' out of range [-99 - 0]
Last message repeated 1 times
[loudnorm @ 0x5610df04ac80] Error setting option measured_thresh to value 10.41.
[Parsed_loudnorm_1 @ 0x5610df044340] Error applying options to the filter.
[AVFilterGraph @ 0x5610defde080] Error initializing filter 'loudnorm' with args 'i=-20.0:lra=7.0:tp=-2.0:offset=0.0:measured_i=0.0:measured_lra=0.0:measured_tp=62.45:measured_thresh=10.41:linear=true:print_format=json'
Error initializing complex filters.
Numerical result out of range

```

**Environment (please complete the following information):**
- ubuntu 18.04
- Python 3.6.7
- ffmpeg version 4.2.1-0york0~18.04.1

**Comment by danisham-afk** [comment]
It can be related to https://github.com/slhck/ffmpeg-normalize/issues/92

**Comment by slhck** [comment]
The file is just 1.2 seconds long, and the measurement is just not accurate for such short lengths. Please see the warnings printed:

> WARNING: Audio stream has a duration of less than 3 seconds. Normalization may not work. See https://github.com/slhck/ffmpeg-normalize/issues/87 for more info.

> WARNING: Input file had measured input loudness greater than zero (input_i), capping at 0

This is, in essence, an upstream issue, but I'm not sure it can be reasonably fixed. You may want to perform RMS-based normalization instead.

**Comment by danisham-afk** [comment]
@slhck I processed thousands such files without any errors.
Only this one is caused the issue. Whole process stucked because of this.
Can you just catch such exceptions and proceed normalize other files from input?

**Comment by danisham-afk** [comment]
Just to clarify, I'm processing the whole directory, not a current file only.

**Comment by slhck** [comment]
Oh, I see. In case of batch conversion, such an exception will stop the whole program. I can look at printing an error for just the affected file, then have the program continue.

**Comment by slhck** [comment]
Fixed an will be available in a few minutes as 1.15.6.

**Comment by danisham-afk** [comment]
@slhck Thank you very much, you are the best maintainer I ever seen
